packs/bit-bucket/actions/list_issues.py

```diff
@@ -0,0 +1,19 @@
+from lib.action import BitBucketAction
+
+
+class ListIssuesAction(BitBucketAction):
+    def run(self, repo):
+        """
+        List Issues of Repository with title
+        of the issue its status and reporter
+        """
+        bb = self.perform_request(repo=repo)
+        success, result = bb.issue.all()
+        issues = {}
+        for i in range(result['count']):
+            issues[result['issues'][i]['title']] = {
```

Comment:
@Kami, Yeah, we can return the whole object, it contains enormous details. I only added the ones that the user would need to further interact with the other actions. Returning the object would be easier for me though.